src/commands/tileindex-validate/__test__/tileindex.validate.test.ts

```diff
@@ -49,14 +49,31 @@ describe('getTileName', () => {
     assert.equal(convertTileName('AT24_50000_0101', 50000), 'AT24');
     assert.equal(convertTileName('AT25_50000_0101', 50000), 'AT25');
     assert.equal(convertTileName('CK08_50000_0101', 50000), 'CK08');
+    assert.equal(convertTileName('CK08_50000_0101', 50000), 'CK08');
```

Comment:
This is identical to the line above. Is that intentional?

Reply:
nope.